Src/CANProcess.c

```diff
@@ -87,7 +110,7 @@ void CANFilterConfig()
 {
 
 
-	  CAN_FilterConfTypeDef FilterConf;
+	  CAN_FilterTypeDef FilterConf;
 	  FilterConf.FilterIdHigh =         ID_PEDALBOX2 << 5; // 2 num
```

Comment:
I know this wasn't part of your change request but we should discuss how we do comments like this. I think it should be inline to the right, but having the different spacing between the comments for 114 and 115 bother me. I'll make a separate issue for this.

Reply:
the formatting has been messed up by copy and pasting so many times + laziness. i will try to go through and clean up these kinds of things